### PR TITLE
Sponsor images

### PR DESCRIPTION
**Pull request by klanmiko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -443,6 +443,41 @@ <h2 class="tracks-title">
...
+        <!-- Sponser images -->
+        <div class="container" data-entrance="fade" data-entrance-delay="100">  
+
+        	<div class="row" style="padding: 1em;">
+	    		<div class="col-sm" style="padding: 2em;">
+	    			<img style="object-fit: contain; max-height: 180px;" class="img-fluid logo-picture" src="sponsors/centene.png">
+	    		</div>
+
+	    		<div class="col-sm" style="padding: 2em;">
+	    			<img style="object-fit: contain; max-height: 180px;" class="img-fluid logo-picture" src="sponsors/twilio.png">
```
Comment (klanmiko) [inline index.html]: these need to be clickable, so each sponsor needs to link to the site @josharnoldjosh

File: index.html [hunk 2]
```diff
@@ -443,6 +443,41 @@ <h2 class="tracks-title">
         <p class="section-title">
           <span class="underline sponsors">sponsors</span>
         </p>
+        <br><br><br>
+
+        <!-- Sponser images -->
+        <div class="container" data-entrance="fade" data-entrance-delay="100">
```
Comment (thomasmunduchira) [inline index.html]: Can we convert the tabs to spaces in your code?

File: index.html [hunk 3]
```diff
@@ -443,6 +443,41 @@ <h2 class="tracks-title">
         <p class="section-title">
           <span class="underline sponsors">sponsors</span>
         </p>
+        <br><br><br>
+
+        <!-- Sponser images -->
+        <div class="container" data-entrance="fade" data-entrance-delay="100">  
+
+        	<div class="row" style="padding: 1em;">
+	    		<div class="col-sm" style="padding: 2em;">
```
Comment (thomasmunduchira) [inline index.html]: Put styling in CSS file instead of inline

**Comment by klanmiko** [comment]
The club finance council has no logo ....

**Comment by klanmiko** [comment]
neither does COSAF

**Comment by klanmiko** [comment]
SISS logo is just the global affairs logo?

**Comment by klanmiko** [comment]
Hey @josharnoldjosh, great set of changes! I added a few style changes to the breakpoints to make sure the sponsors fit on mobile. I think we can put in these changes

**Comment by josharnoldjosh** [comment]
Please see latest commit.

**Pull request merged**